package.json

```diff
@@ -33,24 +33,19 @@
     "client library"
   ],
   "dependencies": {
-    "async": "^2.0.0",
-    "gtoken": "^1.1.0",
-    "lodash.noop": "^3.0.0",
-    "jws": "^3.1.0",
-    "request": "^2.74.0",
-    "string-template": "^1.0.0"
+    "gtoken": "^1.2.1",
+    "lodash.noop": "^3.0.1",
+    "jws": "^3.1.4",
+    "request": "^2.79.0"
```

Comment:
request@2.79.0 has `engines >= 4` in package.json, so `npm install google-auth-library` will at best log a warning, and at worst fail to install due to either the user's own configuration or at runtime if request makes use of a non <4 feature. Do we care?